website/docs/atmo/runnable-api/logging.md

```diff
@@ -23,9 +24,21 @@ log::info(…)
 
 </TabItem>
 
+<TabItem value="js" label="JavaScript/TypeScript 🧪">
+
+In JavaScript and TypeScript the methods live on the `log` import:
+
+```typescript
+import { log } from "@suborbital/runnable"
+
+log.info(…)
+```
+
+</TabItem>
+
 <TabItem value="assemblyscript" label="AssemblyScript 🧪">
 
-In TypeScript/AssemblyScript all methods are prefixed with `log`:
+In AssemblyScript all methods are prefixed with `log`:
 
 ```typescript
```

Comment:
I'd love to clarify: When we're saying this is a `typescript` block, this is AssemblyScript we're showing, right?

Reply:
Just fixed.